.github/workflows/testing.yml

```diff
@@ -0,0 +1,81 @@
+name: macOS tests
+
+on:
+  - push
+  - pull_request
+
+
+jobs:
+  test_macos:
+
+    runs-on: macos-latest
+    strategy:
+      max-parallel: 3
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        numpy-version: ['--upgrade numpy']
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
If possible it would be good for PRs if it built the merged-with-master version (like Travis does) to avoid forcing people to rebase or merge manually to get latest updates/fixes

Reply:
I *think* it's already doing that. See https://github.com/actions/checkout/issues/15

Reply:
Okay great, wasn't sure